src/Core/View.js

```diff
@@ -629,4 +632,96 @@ View.prototype.pickObjectsAt = function pickObjectsAt(mouseOrEvt, ...where) {
     return results;
 };
 
+function viewEvent(type, event, view) {
+    this.type = type;
+    this.viewCoords = view.eventToViewCoords(event).clone();
+    let normalizedCoords;
+    let pickeds;
+    let coordinate;
+    Object.defineProperty(
```

Comment:
For a thing that'll get called every mouse event (so even faster than `requestAnimationFrame` in some case), we shouldn't use defineProperty as it prevents all sort of optimization.

Reply:
So, [the second implementation](
https://github.com/iTowns/itowns/pull/700/files/8d4537380ce0ae02401914a7c212dcae07c5a9f0#diff-d297dcf7b9d0e696ece4b1f6e1d7c69dR675) is better